parameter/Subsystem.cpp

```diff
@@ -104,6 +104,8 @@ bool CSubsystem::needResync(bool bClear)
 // From IXmlSink
 bool CSubsystem::fromXml(const CXmlElement& xmlElement, CXmlSerializingContext& serializingContext)
 {
+    setDescription(getXmlDescriptionAttribute(xmlElement));
```

Comment:
The Element::fromXml is not called because we subsystem does not use the generic fromXml algorithm.